messaging/README.md

```diff
@@ -24,15 +24,15 @@ Getting Started
 1. On the command line run `firebase serve -p 8081` using the Firebase CLI tool to launch a local server.
 1. Open [http://localhost:8081](http://localhost:8081) in your browser.
 1. Click **REQUEST PERMISSION** button to request permission for the app to send notifications to the browser.
-1. Use the generated Instance ID token (IID Token) to send an HTTP request to FCM that delivers the message to the web application, inserting appropriate values for [`YOUR-SERVER-KEY`](https://console.firebase.google.com/project/_/settings/cloudmessaging) and `YOUR-IID-TOKEN`.
+1. Use the generated Instance ID token (IID Token) to send an HTTP request to FCM that delivers the message to the web application, inserting appropriate values for [`YOUR-ACCESS-TOKEN`](https://firebase.google.com/docs/cloud-messaging/auth-server?hl=en&authuser=0#use-credentials-to-mint-access-tokens) and `YOUR-IID-TOKEN`.
```

Comment:
Should "Instance ID token" here be changed to "FCM token"? (based on the comment below https://github.com/firebase/quickstart-js/pull/662/files#r930449480)